winui/Kanban/columns.md

```diff
@@ -1217,6 +1229,8 @@ public class ViewModel
 
 {% endtabs %}
 
+![placeholder-in-winui-kanban](images/columns/placeholder-in-winui-kanban.png)
```

Comment:
Gif not added for drag and drop.

All heading must add image or gif if code added